src/main/resources/init_db.sql

```diff
@@ -7,3 +7,9 @@ CREATE TABLE `manufacturers` (
                                         `country` VARCHAR(225) NOT NULL,
                                         `is_deleted` TINYINT NOT NULL DEFAULT 0,
                                         PRIMARY KEY (`id`));
+CREATE TABLE `drivers` (
+                                        `id` bigint NOT NULL AUTO_INCREMENT,
+                                        `name` varchar(255) DEFAULT NULL,
+                                        `license_number` varchar(255) DEFAULT NULL,
+                                        `is_deleted` tinyint DEFAULT '0',
+                                        PRIMARY KEY (`id`)
```

Comment:
types of var-s should be in upper case))

Reply:
bigint, varchar and tinyint

Reply:
ok

Reply:
agree with previous comment, please rename SQL keywords as uppercase